dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/event/ProgramNotificationInstanceController.java

```diff
@@ -126,7 +126,10 @@ public ProgramNotificationInstanceController(
       return Page.withPager(
           ProgramNotificationInstanceSchemaDescriptor.PLURAL,
           instances,
-          org.hisp.dhis.tracker.export.Page.of(instances, pager, true, false, false));
+          org.hisp.dhis.tracker.export.Page.withTotals(
+              instances, pager.getPage(), pager.getPageSize(), (int) pager.getTotal()),
+          null,
+          null);
```

Comment:
why do we need to pass in `null`? I thought our API in `export.Page` does not require that.

* without totals: no totals and no prev/next
* with totals: total/pageCount but no prev/next
* with prev and next: no totals